_content/development.md

```diff
@@ -0,0 +1,48 @@
+# Web application development
+
+## Web and mobile
+
+We specialise in developing software applications for both Web and mobile using *JavaScript*, *Node.js* and *React*.
+
+We have particular expertise in:
+  
++ Prototyping and UX design
++ Content management
++ Membership and user management
++ Business productivity
++ Data visualisation and mapping
+
+## Agile project mangement
+
+All our projects begin with a period of requirements gathering in which client needs are translated into a series of *user journeys*, *wireframes*, and testable *user stories* that define an application. Development usually proceeds in a series of two-week *sprints*, each involving a planning and review process in which short-term *sprint goals* are set and reviewed. We aim for *continuous delivery* to the client of testable features (*“user stories”*).   
+
+Project teams usually include: a *scrum master*, who ensures that nothing is impeding the work of the development team; a *client liaison*, who ensures that the client is kept fully informed of progress on the project; a *lead developer*, who ensures code quality; and a *product owner*, who takes responsibility for approving user stories as they are delivered. The product owner is usually supplied by the client (for whom we can supply training, if required).
+
+Change requests are added to the *project backlog* by the product owner and incorporated as a matter of course through the use of a repository of new user stories (the *“icebox”*) and a process of re-prioritisation of user stories at the beginning of every sprint. Minor changes are expected and can usually be accommodated within previously-agreed time and budget.
+
+## Test-driven development
+
+We practice *test-driven development* (TDD) and *behaviour-driven development* (BDD), where product features, expressed as user stories, are turned into *acceptance tests*, i.e. automated software tests that demonstrate whether or not the application software actually behaves as expected, and these are written before any code is actually created. We use automated tools to maintain measures of code coverage (i.e. what proportion of the code is covered by an associated test) and to identify failing tests.  
```

Comment:
Do we actually practice BDD? I feel like we do the TDD parts of BDD but we don't actually teach much of the BDD part (http://guide.agilealliance.org/guide/bdd.html)